web_chatter_position/static/src/views/form/form_controller.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates xml:space="preserve">
+    <t t-inherit="web.FormView" t-inherit-mode="extension">
+        <xpath
+            expr="//div[hasclass('o_form_view_container')]/Layout/t[@t-set-slot='control-panel-create-button']/t/button[hasclass('o_form_button_create')]"
+            position="before"
+        >
+          <button
+                type="button"
+                class="btn btn-secondary o_chatter_position_button"
```

Comment:
add class  `d-none d-xxl-inline-block` behind

`class="btn btn-secondary o_chatter_position_button d-none d-xxl-inline-block"`

Reply:
why?

Reply:
to improve the logic of this button from this comment https://github.com/trisdoan/web/pull/7#issuecomment-2563439321

Reply:
No, there is no misunderstanding.

If you think your idea is relevant, please open an PR, thanks

Reply:
hum, I just suggest to avoid unwanted errors. from this [comment](https://github.com/OCA/web/pull/2419#issue-1602900876), you can see one of the reasons he didn't add this button in the migration, probably because he saw the potential of this button when displayed in the size mode that responsive can do. can you imagine what happens when you use your phone to access the form view, see this button and click on it? does Chatter disappear? does that lead to the function of this button being wrong? then why not take this opportunity to block that from happening?